lib/index.js

```diff
@@ -309,7 +320,7 @@ GA.prototype.track = function(track, options) {
     }
   }
 
-  window.ga('send', 'event', payload);
+  window.ga(self._trackerName + 'send', 'event', payload);
```

Comment:
same here, only need to do the `var self = this` if you need to access prototype methods/props inside a closure